vocab/security/template.html

```diff
@@ -362,11 +358,28 @@ <h3>VerificationMethod Section</h3>
           </p>
 
         </section>
+        <section>
+          <h3>Verification Relationship Section</h3>
+          <p>
+            The left side of the section contains a single box, styled as Property
+            and labeled as "proofPurpose". This box is connected to the ellipse
+            "VerificationRelationship" with a connecting line styled as Range, 
+            and is also connected to the ellipse labeled "Proof" in the Proof Section, 
+            with a connecting line styled as Domain Of.
+          </p>
 
-
+          <p>
+            The right side of this Section contains a column of labeled boxes,
+            all styled as Property. The labels, from top to bottom, are
+            "verificationMethod", "authentication", "assertionMethod",
+            "capabilityDelegation", "capabilityInvocation", and "keyAgreement".
+            Each of these boxes is connected to the ellipse labeled "VerificationMethod" in the 
+            Verification section, with a connecting line styled as Range, 
+            and to the ellipse labeled "VerificationRelationship"
+            with a connecting line styled as Type.
```

Comment:
Consistency question (which will lead to some number of changes) — "with a connecting line" should either be always preceded by a comma or always not preceded by a comma. I think there are some instances where the comma helps clarity; I don't *think* there are any where the comma harms clarity.
```suggestion
            Verification section, with a connecting line styled as Range, 
            and to the ellipse labeled "VerificationRelationship"
            with a connecting line styled as Type.
```

Reply:
I agree, and will add the commas.